features/Extended Materials/Shaders/ExtendedMaterials/ExtendedMaterials.hlsli

```diff
@@ -14,14 +14,18 @@ struct DisplacementParams
 
 namespace ExtendedMaterials
 {
-	float AdjustDisplacement(float displacement, DisplacementParams params)
+	float DisplacementHeight0(float displacement, DisplacementParams params)
 	{
-		return (displacement - 0.5) * params.DisplacementScale + 0.5 + params.DisplacementOffset;
+		return (displacement - 0.5) * params.HeightScale + params.DisplacementOffset;
+	}
+	float Displacement01(float displacement, DisplacementParams params)
+	{
+		return (displacement - 0.5) * params.DisplacementScale + params.DisplacementScale / 2 + params.DisplacementOffset;
```

Comment:
Why the change? Pretty sure it was correct before

Reply:
Yeah this might break MLP actually, bad idea.

Reply:
Actually this might work perfectly if DisplacementOffset doesn't exist.

Reply:
That cannot work for the purpose it was introduced: to remap displacement interval for interlayer parallax. So we have `DisplacementScale = 0.5` and `DisplacementOffset = -0.25`, so displacement interval with your formula maps from (0, 1) to (-0.25, 0.25) instead of (0.5, 0.5) since the point is to remap it into negative displacement semi-interval

Reply:
I still don't get this, with these values previously there would be:
(displacement - 0.5) * 0.5 + 0.5 - 0.25
If DisplacementOffset is completely removed and the "+ params.DisplacementScale / 2" is kept, then:
(displacement - 0.5) * 0.5 + 0.25
So there is no difference. Doesn't mean my change makes much sense anyway lol.